lib/kotkowo_web/components/pagination_bar.ex

```diff
@@ -0,0 +1,74 @@
+defmodule KotkowoWeb.Components.PaginationBar do
+  @moduledoc false
+
+  use Phoenix.Component
+
+  attr :selected_page, :integer, required: true
+  attr :last_page, :integer, required: true
+
+  def pagination_bar(assigns) do
+    assigns =
+      assigns
+      |> assign(:previous_page, assigns.selected_page - 1)
+      |> assign(:next_page, assigns.selected_page + 1)
```

Comment:
Don't do any logic here.
Either assign this from the parent or move this to a LiveComponent.
Handle cases where page might be <= 0 and >= max_page

Reply:
Phoenix components are for reusing markup not handling logic.